Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/EncounterTimelineComponent.vue

```diff
@@ -48,24 +48,32 @@ export default class EncounterTimelineComponent extends Vue {
             <b-col>
                 <div class="d-inline-flex" data-testid="encounterClinicLabel">
                     <strong>Clinic/Practitioner:</strong>
-                    <div
+                    <hg-button
+                        v-if="entry.clinic.name"
                         :id="`tooltip-info${index}-${datekey}`"
-                        class="infoIcon ml-2"
-                        tabindex="0"
+                        aria-label="Info"
+                        href="#"
+                        variant="link"
+                        data-testid="health-visit-clinic-name-info-button"
+                        class="shadow-none align-baseline p-0 ml-1"
                     >
-                        <hg-icon icon="info-circle" size="medium" />
-                    </div>
-                    <b-tooltip
-                        variant="secondary"
+                        <hg-icon icon="info-circle" size="small" />
+                    </hg-button>
+                    <b-popover
+                        v-if="entry.clinic.name"
                         :target="`tooltip-info${index}-${datekey}`"
-                        placement="top"
                         triggers="hover focus"
+                        placement="topright"
+                        boundary="viewport"
+                        data-testid="health-visit-location-info-popover"
```

Comment:
`data-testid` should be updated to `health-visit-clinic-name-info-popover` or removed.